### PR TITLE
[IN FCP] RFC 0009 for plan configuration complexity

### PR DESCRIPTION
**Pull request by predominant**
RFC for plan configuration complexity.

### CODE REVIEW

File: _RFCs/0009-plan-configuration-complexity.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+net:
+  port: {{cfg.mongod.net.port}}
+  bindIp: {{cfg.mongod.net.bind_ip}}
+security:
+  authorization: {{cfg.mongod.security.authorization}}
+```
+
+In this case, more options and configurability are provided to the `systemLog` section of the configuration, and can be modified at the users discretion.
+
+Some good ways to consider how apprpriate a configuration is for contribution to core-plans, contributors can reference a couple of sources:
```
Comment (afiune) [inline _RFCs/0009-plan-configuration-complexity.md]: typo `s/apprpriate /appropriate`

File: _RFCs/0009-plan-configuration-complexity.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+A habitat [core-plan] is a plan that is supported and published by core-contributors of the Habitat community. These are often relied upon by other plans, and as part of the Habitat build process itself.
+
+A core-plan's primary responsibility is to ensure the binaries and associated files responsible for running the commands or services are correctly built, packaged and released. In addition, for service pacakges, a minimal configuration representing the minimal/default service configuration is shipped to ensure that services can start in a simple state. This simple/minimal state is representative of the default configuration and minimal configuration required to get the service running.
```
Comment (nellshamrell) [inline _RFCs/0009-plan-configuration-complexity.md]: Slight typo - "pacakges" should be "packages"

**Comment by nellshamrell** [comment]
This definitely has the required amount of approvals, moving into Final Comment Period :)

File: _RFCs/0009-plan-configuration-complexity.md (hunk 2 above)
Comment (predominant) [reply _RFCs/0009-plan-configuration-complexity.md]: Done!

**Comment by predominant** [comment]
Looks like typo fix commits caused all reviews to be cancelled. :(

File: _RFCs/0009-plan-configuration-complexity.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+# Summary
+[summary]: #summary
+
+When packaging software with Habitat, we need to ensure we have an agreed level of complexity with regards to the configuration that is shipped. This RFC proposes a standard by which plans can be developed, and kept simple and functional.
+
+# Motivation
+[motivation]: #motivation
+
+As the [core-plans][core-plans] repository continues to evolve and grow, more users are coming with new and useful use cases for the software that is packaged with Habitat. In order to ensure we maintain the core plans as simple, functional, testable and model examples of software packaging, we must ensure complexity is kept to a minimum. This will encourage simpler configuration in [core-plans][core-plans] plans, and encourage users to use the configuration plan pattern to extend configuration where necessary.
```
[the remark this answers is not in the document]
Comment (robbkidd) [reply _RFCs/0009-plan-configuration-complexity.md]: indeed

**Comment by predominant** [comment]
Thanks! ❤️

**Pull request merged**